### ISSUE TITLE
[libjpeg-turbo] enable jpeg8 features?

### ISSUE THREAD

**Issue by tomty89** [termux/termux-packages#1891]
For that `--with-jpeg8` needs to be stated explicitly to `configure`.

Any reason not to do it in Termux? FYR, Ubuntu built libjpeg-turbo with that and additionally built libjpeg6 and libjpeg9 from ijs. In Arch none of the ijs libjpeg is provided, and the libjpeg-turbo build used by basically every package is built with that.

P.S. Not sure if packages that dep on libjpeg-turbo needs to rebuilt afterwards

**Comment by fornwall** [comment]
We can probably build with `--with-jpeg8` (and AFAIK we would need to revision-bump and rebuild all packages linking against libjpeg, as it changes the ABI).

For my understanding: Which advantages does building with `--with-jpeg8` have for you?

**Comment by tomty89** [comment]
To be frank I only happened to notice it because it is the dependency of the package I recently requested (qpdf). My concern was without it enabled, the images produced by the programs that depends on libjpeg-turbo would be slightly inferior than those produced on a desktop distro. I am not sure how true that was though.

P.S. One thing I noticed is that, its own `jpegtran` would support lossless cropping with this enabled.
